### DISCUSSION TITLE
post/xperia-5-ii/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#28] [Announcements]
# post/xperia-5-ii/

之前的红米 Note 10 Pro 5G (chopin) 在我手下服役近两年, 产出(水)了若干篇博客后[1, 2, 3], 甚至成功运行了 Windows 11. 不过它在近期频繁出现音量键和充电问题, 加之社区开发者逐渐流失, MediaTek SoC 的它运行安卓 14 还有蓝牙和内存泄漏问题, 于是打算换机. 还能再水一篇博客. 选购”新”机我的手机选择标准:  必须项目: 能够解锁 B

https://blog.lyc8503.net/post/xperia-5-ii/

**Comment by izfsk-ium** [comment]
相当于是手机当 Chrome Book 用了，酷！

**Comment by MousseC** [comment]
5ii的绿紫线机太通病了，索尼买的三星屏幕的那个产品质量真的垃圾，只要你用原厂屏幕一定会出现。我绿线之后换成1V了，旧机在想要不要去换个国产的LCD屏幕……
